extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+---
+
+### content
+
+Provides information about the content of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**levelOffset**|`number`|An integer describing which level the content bitsream, if provided, starts at|No|
+|**levelOffsetFill**|`number`|An integer describing the state value for the content bit for all levels up to the `levelOffset`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the content buffer.|No|
+
+---
+
+### metadata
+
+Provides information about the metadata of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**levelOffset**|`number`|An integer describing which level the metadata bitsream, if provided, starts at|No|
+|**levelOffsetFill**|`number`|An integer describing the state value for the metadata bit for all levels up to the `levelOffset`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the metadata buffer.|No|
+
+## Appendix
+
+### Algorithms
+
+#### Jump Buffer Construction
+
+```javascript
+function createJumpBuffer(sBuffer) {   
+    let jBuffer = [];
+    let sBufferIndex = 1;
+    let currentLevel = 0;
+    while (sBufferIndex != sBuffer.length) {
+        currentLevel++;
+        jBuffer.push([]);
+        // Get maximum number of elements in this level, based on number of tiles in the level above that will subdivide.
+        let currentLevelMax = jBuffer[currentLevel - 1].length * tilingScheme;
+        jumpBufferSize += currentLevelMax;
+        for (let i = 0; i < currentLevelMax; i++) {
+            if (sBuffer[sBufferIndex++] === 1) {
+                jBuffer[currentLevel].push(jBuffer[currentLevel - 1][Math.floor(i / tilingScheme)] * tilingScheme + (i % tilingScheme));
+            }
+        }
+    }
+    return jBuffer;
+}
+```
+
+#### Tile Index Lookup
+
+```javascript
+function getIndex(level, x, y, z) {
+    const mortonIndex = morton(x, y, z);
+    if (level === 1) {
+        return 1 + mortonIndex;
+    }
+    // Level 1 Search
+    const levelOneCellMorton = mortonIndex >> 2;
+    if (sBuffer[1 + levelOneCellMorton] === 1) {
+        // Level 2... Search
+        return traverse(level, mortonIndex, 2, 5);
+    }
+    return -1;
+}
+
+function traverse(targetLevel, morton, currentLevel, levelOffset) {
+    const parentTileMorton = morton >> (2 ^ (targetLevel - currentLevel + 1));
+    const currentTileOffset = morton % 4;
+    const currentTileBufferOffset = levelOffset + jBuffer[currentLevel - 1].indexOf(parentTileMorton) + currentTileOffset;
+
+    if (targetLevel === currentLevel) {
+        return currentTileBufferOffset;
+    }
+
+    if (sBuffer[currentTileBufferOffset] === 1) {
+        return traverse(targetLevel, morton, currentLevel + 1, levelOffset + (jBuffer[currentLevel].length * tilingScheme));
+    } else {
+        return -1;
+    }
+}
+```
+
+
+
+
+### Samples
+
+#### Complete Quadtree
+
+##### Base Tileset - tileset.json
```

Comment:
As we talked about offline, let's present the examples like this:

1. Complete quadtree as a single tileset
2. Double-headed quadtree as two separate tilesets
3. Double-headed quadtree with a base explicit tileset
4. Six-headed S2-like example with base explicit tileset

Reply:
Added examples along with directory structure.